src/registry/index.ts

```diff
@@ -0,0 +1,131 @@
+import { type SignerConfig } from "../helpers/connection.js";
```

Comment:
Public interface of the registry class. Most folks won't even use this, but if they want finer grained control over how they interact with the tableland network, they have some exposed methods here. We can easily expose more as needed.